rwatch/ui/layer/menu_layer.c

```diff
@@ -102,13 +102,28 @@ static bool has_next_index(const MenuLayer *menu_layer, const MenuIndex *index,
            || (!up && (index->section < menu_layer->end_index.section || index->row < menu_layer->end_index.row - 1));
 }
 
-static MenuIndex get_next_index(MenuLayer *menu_layer, bool up)
+static MenuIndex get_next_index(MenuLayer *menu_layer, ClickRecognizerRef ref, bool up)
```

Comment:
Architectural nit: I would probably make this a `bool do_wrap` that gets passed through to this layer (similar to the `bool up`), rather than passing the `ClickRecognizerRef` through.